tests/api/test_superadmin_only.py

```diff
@@ -629,13 +629,26 @@ def test_set_maintenance_not_superadmin(client: flask.testing.FlaskClient) -> No
         assert response.status_code == http.HTTPStatus.FORBIDDEN
 
 
-def test_set_maintenance_incorrect_method(client: flask.testing.FlaskClient) -> None:
-    """Only put should be accepted."""
+def test_get_maintenance_status_not_superadmin(client: flask.testing.FlaskClient) -> None:
+    """Check Maintenance mode status using everything but Super Admin access."""
+    no_access_users: typing.Dict = users.copy()
+    no_access_users.pop("Super Admin")
+
+    for u in no_access_users:
+        token: typing.Dict = get_token(username=users[u], client=client)
+        response: werkzeug.test.WrapperTestResponse = client.get(
+            tests.DDSEndpoint.MAINTENANCE, headers=token, json={"state": "status"}
```

Comment:
I don't believe this is needed? 
```suggestion
            tests.DDSEndpoint.MAINTENANCE, headers=token
```